care/facility/migrations_old/0073_auto_20200404_2303.py

```
@@ -1,3 +1,4 @@
+from django.utils.translation import gettext_lazy as _
```

Ruff (0.8.2)

Invalid module name: '0073_auto_20200404_2303'

(N999)